<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoLister</class>
 <widget class="QWidget" name="PhotoLister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>418</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Photos List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/photo.png</normaloff>:/images/photo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidgetPhotos">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnAddPhoto">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemovePhoto">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/del.png</normaloff>:/images/del.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemovePhotos">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Erase List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNbPhotos">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0 Photos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddPhotoFromComputer">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/computer_download.png</normaloff>:/images/computer_download.png</iconset>
   </property>
   <property name="text">
    <string>Add Photo From Computer...</string>
   </property>
  </action>
  <action name="actionAddPhotoFromWeb">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/web_download.png</normaloff>:/images/web_download.png</iconset>
   </property>
   <property name="text">
    <string>Add Photo From Web...</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Left</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Right</string>
   </property>
  </action>
  <action name="actionRemoveSelectedPhotos">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/del.png</normaloff>:/images/del.png</iconset>
   </property>
   <property name="text">
    <string>Remove Selected Photos</string>
   </property>
  </action>
  <action name="actionRemoveAllPhotos">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/del.png</normaloff>:/images/del.png</iconset>
   </property>
   <property name="text">
    <string>Remove All Photos</string>
   </property>
  </action>
  <action name="actionSelectAllPhotos">
   <property name="text">
    <string>Select All Photos</string>
   </property>
  </action>
  <action name="actionPreviewPhoto">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/viewer.png</normaloff>:/images/viewer.png</iconset>
   </property>
   <property name="text">
    <string>Preview Photo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
